frontend/src/components/FooTouer/FooTouer.css

```diff
@@ -0,0 +1,3 @@
+body {
+    margin: 0;
+}
```

Comment:
hę? Opisz mi proszę co tutaj się dzieje - tzn.  dlaczego tu akurat jest wrzucone margin body i tylko (albo aż) to

Reply:
To są style przeniesione ze starego pliku index.css, które są defaultowo ustawione przy instalacji projektu. Usunęłam wszystko co dotyczyło wyświetlania interaktywnego logo reacta (style czcionki i logo), zostawiłam tylko tyle co widać. Mogę to usunąć

Reply:
Ja bym zostawił. Inaczej default owo jest margin całej strony i nie możesz ustawić np. Tła. Ale może to i być elementem taska jakiegoś jak ktoś chce. Nie zatrzymywał bym się teraz nad tym dluzej :)

Reply:
Kurde, na prośbę Piotrka właśnie to usunęłam :)

Reply:
@nowakprojects , ale piszę to do wrzucenia do index.css